core_types/Cargo.toml

```diff
@@ -18,6 +18,7 @@ bitflags = "1.0"
 objekt="0.1.1"
 holochain_core_types_derive = { path = "../core_types_derive" }
 uuid = { version = "0.7", features = ["v4"] }
+regex="1.1.0"
```

Comment:
Please put spaces around the equal sign. Makes it a little easier to read

Reply:
Also don't make the version completely strict to avoid crate duplication. We can probably make it `^1.1`